homeassistant/components/drop/strings.json

```diff
@@ -0,0 +1,45 @@
+{
+  "config": {
+    "abort": {
+      "not_supported": "Configuration for DROP is through MQTT discovery.  Use the DROP Connect app to connect your DROP Hub to your MQTT broker."
```

Comment:
```suggestion
      "not_supported": "Configuration for DROP is through MQTT discovery. Use the DROP Connect app to connect your DROP Hub to your MQTT broker."
```

Reply:
I'm old enough that two spaces after a period was once required.  Can't seem to shake it. ;)  Addressed in 35b94a287d99d6ac74f337b4b8d1b8a83dcbcfd0.